pkg/costmodel/assets.go

```diff
@@ -133,6 +133,14 @@ func (cm *CostModel) ComputeAssets(start, end time.Time) (*kubecost.AssetSet, er
 		node.GPUCost = n.GPUCost
 		node.GPUCount = n.GPUCount
 		node.RAMCost = n.RAMCost
+		combinedRamCPUCost := n.CPUCost + n.RAMCost
+
+		node.Overhead = &kubecost.NodeOverhead{
+			RamOverheadFraction: n.Overhead.RamOverheadFraction,
+			CpuOverheadFraction: n.Overhead.CpuOverheadFraction,
+			OverheadCostFraction: (n.Overhead.CpuOverheadFraction * (n.CPUCost / combinedRamCPUCost)) +
+				(n.Overhead.RamOverheadFraction * (n.RAMCost / combinedRamCPUCost)),
+		}
```

Comment:
I think this can be inaccurate for node with GPUs (or any extra associated cost). Something like this should take in into account:

```go
OverheadCostFraction: (n.Overhead.CpuOverheadFraction * n.CPUCost  +
				n.Overhead.RamOverheadFraction * n.RAMCost) / n.TotalCost,
```

Reply:
fixed!